ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -452,21 +455,25 @@ private class JdbcLedgerDao(
 
   override def storeTransactionState(
       preparedInsert: PreparedInsert
-  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
+  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] = {
+    logger.info("Storing transaction state")
```

Comment:
We can skip this log, the final one when completing is enough

Reply:
Same comment as [above](https://github.com/digital-asset/daml/pull/8676#discussion_r571638038).